templates/commands/render_test.go

```diff
@@ -185,6 +185,28 @@ steps:
     paths: ['file1.txt', 'dir1', 'dir2/file2.txt']
 `
 
+	inputsSpecContents := `
```

Comment:
Can we add the inputs to `specContents` and avoid the need to create `inputSpecContents`?

Reply:
ack